README.md

```diff
@@ -1,6 +1,581 @@
 # plasmactl-bump
-Update the version of components that were updated in last commit
 
-### Excluded files
+Update the version of components that were updated in the last commit.
+
+### Excluded Files:
+
 - README.md
 - README.svg
+
+---
+
+**bump:** Updates the version of components that were modified after the last bump.
+
+### Bump flow
```

Comment:
This section is quite similar to `### Overall Propagation Workflow:` > let's merge both